components/mediation/security/org.wso2.micro.integrator.mediation.security/src/main/java/org/wso2/micro/integrator/mediation/security/vault/SecureVaultConstants.java

```diff
@@ -42,4 +44,20 @@
 	String CONF_CONNECTOR_SECURE_VAULT_CONFIG_PROP_LOOK =
 	                                                                          "conf:/repository/components/secure-vault";
 
+	/**
+	 * System property to specify customized docker secret root directory
+	 */
+	public static final String PROP_DOCKER_SECRET_ROOT_DIRECTORY = "ei.secret.docker.root.dir";
+	public static final String PROP_DOCKER_SECRET_ROOT_DIRECTORY_DEFAULT = "/run/secrets/";
+	public static final String PROP_DOCKER_SECRET_ROOT_DIRECTORY_DEFAULT_WIN = "C:\\ProgramData\\Docker\\secrets";
```

Comment:
People can customize the docker installation. In that case, don't we need to make these configurable?